lib/pages/tabs/bmm_tab_page.dart

```diff
@@ -5,189 +5,163 @@ import 'dart:math';
+                width: 250,
+                child: SailTextField.tiny(
+                  controller: viewModel.refreshController,
+                  hintText: 'Refresh interval',
+                  prefix: 'Refresh interval: ',
+                  suffix: 'seconds',
+                  textFieldType: TextFieldType.number,
+                ),
+              ),
+              Expanded(child: Container()),
+            ],
+          ),
+          body: Center(
+            child: Column(
+              children: [
                 const SizedBox(height: 20),
                 viewModel.bmmAttempts.isEmpty
                     ? Padding(
                         padding: const EdgeInsets.all(40),
-                        child: SailText.primary20(
-                          'No BMM attempts yet. Click the start button!',
-                          bold: true,
+                        child: SailColumn(
+                          spacing: SailStyleValues.padding15,
+                          children: [
+                            ConstrainedBox(
+                              constraints: const BoxConstraints(
+                                maxWidth: 640,
+                              ),
+                              child: SailText.primary13(
+                                'Blind Merged Mining (BMM) is the mechanism used for mining on a sidechain.\n\n'
+                                'Pressing the "start" button here starts the BMM process. The BMM process consists '
+                                'of a sequence of attempts which can succeed or fail. When an attempt is first made '
+                                'it has a pending status. The next attempt decides the outcome of the previous one, '
+                                'which is reflected in the table below.\n\n'
+                                'A BMM attempt can also fail to be created in its entirety. In that case, it is not '
+                                "added to the table. Notably, a BMM attempt only succeeds if there's been mined at"
+                                'least 1 block on the mainchain since the last BMM attempt.\n\n'
+                                "A BMM attempt includes a satoshi denominated 'bid' amount. This amount is paid to "
+                                'the the mainchain miners for the right to construct a new sidechain block. The '
+                                'profit from mining the sidechain block is the total fees collected on the sidechain, '
+                                'minus the bid to the mainchain miners. If the fees collected on the sidechain is '
+                                'less than the bid amount, the profit goes negative (money lost!)'
+                                '',
+                              ),
+                            ),
+                            SailText.primary20(
+                              'No BMM attempts yet. Click the start button!\n(and automine at the same time)',
```

Comment:
This assumes that the mainchain is started from the launcher. Could potentially add an option to automine from Sidesail?

Reply:
How can we do that?

Reply:
I agree that is a very nice UX, but can we hold off a bit on UX benefits like that until everything is implemented?

next iteration kinda stuff

Reply:
We should keep adding these UX things that pop up as we go, but keep the scope as small as possible early on imo.

Can keep track of these things in linear, and add them to a later project iteration